aries-site/src/examples/components/maskedinput/MaskedIPRangeExample.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { Form, FormField, MaskedInput } from 'grommet';
+
+const IPv4ElementExp = /^[0-1][0-9][0-9]$|^2[0-4][0-9]$|^25[0-5]$|^[0-9][0-9]$|^[0-9]$/;
+
+export const MaskedIPRangeExample = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <Form>
+      <FormField
+        label="IP Range"
+        htmlFor="masked-ip-range"
+        name="maskedIpRange"
+      >
+        <MaskedInput
+          name="maskedIpRange"
+          id="masked-ip-range"
+          mask={[
```

Comment:
Suggestion: separate the mask logic from the MaskInput to aid readability and cognition.